docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,193 @@
+---
+id: reanimated-drawer-layout
+title: Reanimated Drawer Layout
+sidebar_label: Reanimated Drawer Layout
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cross-platform replacement for the React Native's [DrawerLayoutAndroid](http://reactnative.dev/docs/drawerlayoutandroid.html) component.  
+Provides a compatible API while allows for the component to be used on both Android and iOS.  
+Please refer to [React Native docs](http://reactnative.dev/docs/drawerlayoutandroid.html) for the detailed usage for standard parameters.
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import ReanimatedDrawerLayout from 'react-native-gesture-handler/ReanimatedDrawerLayout';
+```
+
+## Properties:
+
+### `drawerType`
+
+accepts values of the `DrawerPosition` enum, defaults to `DrawerPosition.FRONT`.  
+Specifies the way the drawer will be displayed.
+
+- `DrawerPosition.FRONT` the drawer will display on top of the content view.
+- `DrawerPosition.BACK` the drawer appears to be behind the content view, revealed by sliding away the content view.
+- `DrawerPosition.SLIDE` the drawer displays attached to the content view, opening it slides both the drawer and the content view.
+
+| `DrawerPosition.FRONT`                            | `DrawerPosition.BACK`                            | `DrawerPosition.SLIDE`                            |
+| ------------------------------------------------- | ------------------------------------------------ | ------------------------------------------------- |
+| <img src={useBaseUrl('gifs/drawer-front.gif')} /> | <img src={useBaseUrl('gifs/drawer-back.gif')} /> | <img src={useBaseUrl('gifs/drawer-slide.gif')} /> |
+
+### `edgeWidth`
+
+number. Width of the invisible draggable area on the edge of the content view, which can be dragged to open the drawer.
```

Comment:
I know that it probably would require more changes in our docs, but I don't like the fact that the line starts with _**number.**_

I think that we could either include type in header (and that requires a bit more work in docs), or do something similar to [this](https://docs.swmansion.com/react-native-gesture-handler/docs/components/reanimated_swipeable#friction) or [that](https://docs.swmansion.com/react-native-gesture-handler/docs/components/reanimated_swipeable#friction).

Reply:
I understand, fixed this and other instances of types being bluntly stated in [a5e9409](https://github.com/software-mansion/react-native-gesture-handler/pull/3188/commits/a5e9409701e45d9c7961bcec2b0a83e07a79ae43)